packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -112,3 +112,11 @@ Template locking allows locking the `InnerBlocks` area for the current template.
 If locking is not set in an `InnerBlocks` area: the locking of the parent `InnerBlocks` area is used.
 
 If the block is a top level block: the locking of the Custom Post Type is used.
+
+### `placeholderType`
+* **Type:** `String`
+* **Default:** - uses the `DefaultBlockAppender` (`packages/block-editor/src/components/default-block-appender/index.js`) to automatically insert a `paragraph` Block. 
```

Comment:
It doesn't necessarily insert a paragraph. It inserts whichever block is configured as the default block via `wp.blocks.getDefaultBlockName`. It could be fair to imply that it's usually a paragraph, though it is possible for this to be customized by plugins.

Reply:
I've updated the docs to reflect this plus the changes I've just made.